packages/design-tokens/src/scss/generated/_gradients.scss

```diff
@@ -1,10 +1,14 @@
 // Generated Gradients from Supernova. Do not edit manually.
 $gradient-background-basic-overlay: linear-gradient(var(--angle, 90deg), #fff 0%, #ffffff00 100%) !default;
+$gradient-blue: linear-gradient(var(--angle, 180deg), #9ab3db 0%, #3f74c9 100%) !default;
```

Comment:
We've discussed about better naming for this category of the gradients but it's one of the toughest part of our jobs. :) Jirka B. would like to solved it in the future after a consultation with Spirit.